packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -273,6 +284,15 @@ const GeneralTab = decorate([
         <GuestToolsDetection vm={vm} />
         {/* TODO: use CSS style */}
         <br />
+        <Row className='text-xs-center'>
+          <Col>
+            <p>
+              {_('secureBootStatus')}
+              {': '}
+              {SECUREBOOT_STATUS_MESSAGES[vmSecurebootReadiness]}
```

Comment:
Use the `keyValue` message and pass it the key `_('secureBootStatus')` and the value `SECUREBOOT_STATUS_MESSAGES[vmSecurebootReadiness]`.